node/nodejs_intro/README.md

```diff
@@ -169,6 +231,21 @@ What do we think the output to screen will be now?
   <summary>
     Solution
   </summary>
-   The output will be identical to the previous example. The reason: Our setTimeout is moved into the WebAPI regardless of
-  how quickly it is set to resolve.
+
+
+
+    Hello,
+    Yoda
+    I am
+
+   The output will be identical to the previous example. The reason: Regardless, of how quickly our setTimeout is set to resolve (in this case, in 0 seconds), it is still moved into the WebAPI.
 </details>
+<p>&nbsp;</p>
+
```

Comment:
Would love to see a few more problems at the end here, with examples of code similar to the Yoda example where we ask learners to a) predict what the output will be and b) describe what is happening with each line of code in terms of the event loop.
I think it's fair to say that many folks will need a good amount of repetition with this concept before they start to get it down.

Reply:
👍